build.gradle

```diff
@@ -366,66 +350,60 @@ signing {
+            }
 
-                pom.project {
-                    name 'Hellbender'
-                    packaging 'jar'
-                    description 'Development on GATK4'
-                    url 'http://github.com/broadinstitute/hellbender'
+            pom.project {
+                name 'GATK4'
+                packaging 'jar'
+                description 'Development on GATK4'
+                url 'http://github.com/broadinstitute/hellbender'
 
-                    scm {
-                        url 'scm:git@github.com:broadinstitute/hellbender.git'
-                        connection 'scm:git@github.com:broadinstitute/hellbender.git'
-                        developerConnection 'scm:git@github.com:broadinstitute/hellbender.git'
-                    }
+                scm {
+                    url 'scm:git@github.com:broadinstitute/hellbender.git'
+                    connection 'scm:git@github.com:broadinstitute/hellbender.git'
+                    developerConnection 'scm:git@github.com:broadinstitute/hellbender.git'
+                }
 
-                    developers {
-                        developer {
-                            id = "gatkdev"
-                            name = "GATK Development Team"
-                            email = "gatk-dev-public@broadinstitute.org"
-                        }
+                developers {
+                    developer {
+                        id = "gatkdev"
+                        name = "GATK Development Team"
+                        email = "gatk-dev-public@broadinstitute.org"
                     }
+                }
 
-                    licenses {
-                        license {
-                            name 'BSD 3-Clause'
-                            url 'https://github.com/broadinstitute/hellbender/blob/master/LICENSE.TXT'
-                            distribution 'repo'
-                        }
+                licenses {
+                    license {
+                        name 'BSD 3-Clause'
+                        url 'https://github.com/broadinstitute/hellbender/blob/master/LICENSE.TXT'
+                        distribution 'repo'
                     }
                 }
             }
         }
-    } else {
-        doFirst({
-            logger.error( 'Users are not generally supposed to upload archives.  ' +
-                'To upload archives you must specify certain information in your gradle.properties. See the README' +
-                ' for more information.' )
-            throw new Exception("Missing sonatype username or password, please see the README for properties that must be specified for uploading an archive.")}
-        )
     }
+}
+
 
 
 task installSpark{ dependsOn sparkJar }
 task installAll{  dependsOn installSpark, installDist }
 
 test.dependsOn extractIntelDeflater
 installDist.dependsOn downloadGsaLibFile, extractIntelDeflater
-build.dependsOn installDist
-check.dependsOn installDist
```

Comment:
They just didn't really do what you'd want.  There's no reason that check should depend on installDist.  We could make build do installAll, but that's what we have installAll for...